guides/source/upgrading_ruby_on_rails.md

```diff
@@ -324,14 +324,14 @@ Active Record Encryption now uses SHA-256 as its hash digest algorithm. If you h
 versions, there are two scenarios to consider:
 
 1. If you have `config.active_support.key_generator_hash_digest_class` configured as SHA-1 (the default
-   before Rails 7.0), you need to configure SHA-1 for Active Record Encryption too:
+   before Rails 7.1), you need to configure SHA-1 for Active Record Encryption too:
```

Comment:
https://guides.rubyonrails.org/configuring.html#config-active-support-key-generator-hash-digest-class

It seems more nuanced that that... if an app was generated with 7.0 shouldn't it be on `SHA-256`?